tests/package.json

```diff
@@ -52,7 +52,7 @@
     "watch": "npm-watch",
     "build": "npx tsc && cp -r contracts build/",
     "build-clean": "rm -r node_modules && npm i && npm run build",
-    "test-single": "mocha -r ts-node/register 'tests/test-xcm/test-xcm-erc20-transfer.ts'",
+    "test-single": "mocha -r ts-node/register 'tests/test-staking/test-mark-offline.ts'",
```

Comment:
I'm quite surprised that we have this npm run action :P 

You can use `npm run test -- --grep 'Test Describe Name'` to run individual tests to avoid committing this file.

Also, perhaps you forgot to commit the file `test-mark-offline.ts` ? :D